src/components/Header/Header.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { PureHeader, Header } from './Header';
+
+describe('PureHeader', () => {
+  it('renders', () => {
+    const menu = [{ label: "testlabel1", path: "/testlabel1" }]
+    const data = { site: { siteMetadata: { title: 'Test Title', menu } } };
+    const { container } = render(<PureHeader data={data} />);
+    expect(container).not.toBeNull();
+  })
+
+  it('renders menu correctly', () => {
+    const menu = [{ label: "testlabel1", path: "/testpath1" }, { label: "testlabel2", path: "/testpath2" }];
+    const data = { site: { siteMetadata: { title: 'Test Title', menu } } };
+    const { container, queryAllByText } = render(<PureHeader data={data} />);
+    expect(container.querySelectorAll('.Header-link')).toHaveLength(3);
+    expect(container.querySelectorAll('link[to="/testpath1"]')).toHaveLength(2);
```

Comment:
Instead of dom selection, is it possible to find the link components with getByText [seen here](https://testing-library.com/docs/dom-testing-library/api-queries#textmatch-examples). 

Doing so gets us closer to what the user would actually be doing, and makes this test more resilient to change of the underlying components.

Reply:
At the moment i haven't been able to find a way to test the paths without using querySelectors and using any getBy* method, but anyways still looking for a solution on this